spyder/widgets/sourcecode/editorextensions/manager.py

```diff
@@ -16,17 +17,16 @@
 
 
 class EditorExtensionsManager(Manager):
-    """
-    Manages the list of editor extensions of the CodeEdit widget.
-    """
+    """Manages the list of editor extensions of the CodeEdit widget."""
 
     def __init__(self, editor):
+        """Initialize and add a reference to the editor."""
         super(EditorExtensionsManager, self).__init__(editor)
         self._extensions = {}
 
     def append(self, extension):
```

Comment:
Yes it is, but append even your docsting uses ADD and not APPEND, so you can see why append is not really a good choice, independent of the structure used to store extensions (dict, list, set or whatever). So either we append/prepend or add/remove, insert/pop, but not a mix of those. So yes, please change to `add`